mlir/lib/IR/OperationSupport.cpp

```diff
@@ -696,7 +705,9 @@ llvm::hash_code OperationEquivalence::computeHash(
     Region *lhs, Region *rhs,
     function_ref<LogicalResult(Value, Value)> checkEquivalent,
     function_ref<void(Value, Value)> markEquivalent,
-    OperationEquivalence::Flags flags) {
+    OperationEquivalence::Flags flags,
+    function_ref<LogicalResult(ValueRange, ValueRange)>
+        checkCommutativeEquivalent) {
```

Comment:
`checkCommutativeEquivalent` is not used in this function.

Reply:
Good catch, forgot to pass it in below.